app/components/text-currency/text-currency.tsx

```diff
@@ -31,17 +31,17 @@ export const TextCurrencyForAmount: React.FC<Props> = ({
   iconColor,
 }) => {
   const { formatToDisplayCurrency } = useDisplayCurrency()
```

Comment:
Is this component still used anywhere?

Reply:
Yeah still in balance header
Plan to removed it when I'm doing a PR for it next

Reply:
Sounds good!